src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/draftorders/upload/UploadDraftOrdersSubmittedHandlerTest.java

```diff
@@ -139,6 +148,30 @@ private static List<SuggestedDraftOrderCollection> suggestedOrdersCollection(
             .toList();
     }
 
+    @Test
+    void shouldClearTemporaryFieldsAfterHandling() {
+        String caseReference = "1727874196328932";
+        FinremCaseData caseData = FinremCaseData.builder()
+            .draftOrdersWrapper(DraftOrdersWrapper.builder()
+                .uploadSuggestedDraftOrder(UploadSuggestedDraftOrder.builder().build())
+                .uploadAgreedDraftOrder(UploadAgreedDraftOrder.builder().build())
+                .agreedDraftOrderCollection(agreedDraftOrdersCollection(
+                    List.of(LocalDateTime.now())))
+                .build())
+            .build();
+
+        doNothing().when(notificationService).sendContestedOrderReadyToReviewToJudge(any(FinremCaseDetails.class));
+
+        FinremCallbackRequest request = FinremCallbackRequestFactory.from(Long.parseLong(caseReference),
+            CaseType.CONTESTED, caseData);
+
+        var response = uploadDraftOrdersSubmittedHandler.handle(request, AUTH_TOKEN);
+
+        assertThat(response.getConfirmationHeader()).isEqualTo("# Draft orders uploaded");
```

Comment:
why don't you separate it to another method? or we should change the junit method name.